app/script/components/userList.js

```diff
@@ -75,9 +75,9 @@ z.components.UserList = class UserList {
           const trimmedQuery = this.filter().trim();
           const isHandle = trimmedQuery.startsWith('@') && z.user.UserHandleGenerator.validate_handle(normalizedQuery);
           const excludedEmojis = Array.from(normalizedQuery).filter(char => EMOJI_UNICODE_RANGES.includes(char));
-          return this.userEntities().filter(userEntity =>
-            userEntity.matches(normalizedQuery, isHandle, excludedEmojis)
-          );
+          return this.userEntities().filter(userEntity => {
+            return userEntity.matches(normalizedQuery, isHandle, excludedEmojis);
```

Comment:
Why more overhead?

Reply:
It's not overhhead. If we do not do an explicit return in this case you need to read two lines to see that something is returned due to it being split into multiple lines by prettier. With this you immediately see what's happening.